formstack-baton-requests/src/local/com/gu/identity/formstackbatonrequests/PopulateFormstackTable.scala

```diff
@@ -0,0 +1,20 @@
+package com.gu.identity.formstackbatonrequests
+
+import com.gu.identity.formstackbatonrequests.aws.{Dynamo, SubmissionTableUpdateDate}
+import com.typesafe.scalalogging.LazyLogging
+
+/* Script used to populate Formstack table from empty. When .updateDynamo is not provided a date, Dynamo will be
+ * updated with ALL submissions from Formstack */
+object PopulateFormstackTable extends App with LazyLogging {
```

Comment:
what happens if we had entries in dynamo table and we run this object? Would this new run delete all entries and add new ones or it just updated entries?

Reply:
If we ran this object, it would go through all entries again but because the key is email, submission id, it will overwrite any duplicates I think

Reply:
okay and updating these would be fine for us without any consequences?

Reply:
Yes, should be fine. We should only ever need to run this script once but I thought I might as well keep it in here in case we ever need to remove the table and rerun it